types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts

```diff
@@ -1,27 +1,21 @@
-import {
-    Variables,
-    NormalizationField,
-    NormalizationOperation,
-    NormalizationSelection,
-    NormalizationLinkedField,
-    NormalizationScalarField,
-    OperationDescriptor,
-    GraphQLResponse,
-    NormalizationSplitOperation,
-} from 'relay-runtime';
+import { GraphQLSingularResponse, OperationDescriptor } from 'relay-runtime';
 
-interface MockResolverContext {
-    parentType?: string | undefined;
-    name?: string | undefined;
-    alias?: string | undefined;
-    path?: ReadonlyArray<string> | undefined;
-    args?: { [key: string]: any } | undefined;
+export interface MockResolverContext {
+    parentType?: string | null | undefined;
+    name?: string | null | undefined;
+    alias?: string | null | undefined;
+    path?: ReadonlyArray<string> | null | undefined;
+    args?: { [key: string]: any } | null | undefined;
 }
-type MockResolver = (context: MockResolverContext, generateId: () => number) => unknown;
+
+export type MockResolver = (context: MockResolverContext, generateId: () => number) => unknown;
+
 export interface MockResolvers {
     [typeName: string]: MockResolver;
 }
 
-export function generate(operation: OperationDescriptor, mockResolvers?: MockResolvers): GraphQLResponse;
-
-export {};
+export function generate(
+    operation: OperationDescriptor,
+    mockResolvers?: MockResolvers | null,
```

Comment:
I think add `| undefined` to this line and next

Reply:
the lint rule prohibits this;

```
ERROR: 19:44  redundant-undefined  Parameter is optional, so no need to include `undefined` in the type. See: https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/dtslint/docs/redundant-undefined.md
```